src/shogun/base/SGObject.cpp

```diff
@@ -829,10 +806,7 @@ void CSGObject::list_observable_parameters()
 	SG_PRINT("------");
 	for (auto const& x : param_obs_list->get_list())
 	{
-		SG_PRINT(
-		    "%s [%s]: %s\n", x.first.c_str(),
-		    param_obs_list->type_name(x.second.first).c_str(),
-		    x.second.second.c_str());
```

Comment:
could we remove this method and instead make one that returns `std::vector<std::string>` like we recently did for parameter names?

Reply:
It is fine with me. It will be more consistent with the general Shogun's behaviour.